source/sysadmins-guide/shopware-5-performance-for-sysadmins/index.md

```diff
@@ -17,7 +17,7 @@ menu_title: Performance Guide
 menu_order: 50
 ---
 
-In this document we detail performance related settings that you can set in your server to get the most out of it. Some of them were already part of previous Shopware releases, which we complemented with new addictions, for optimized performance and scalability. In most cases, it's assumed that you have already installed and configured Shopware on your server, and that it's running properly. This document does not cover configuration options *needed* by Shopware (for example, increasing PHP's `memory_limit` to an acceptable level), and focus only on making an already working system perform better.
+In this document we detail performance related settings that you can set in your server to get the most out of it. Some of them were already part of previous Shopware releases, which we complemented with new additions, for optimized performance and scalability. In most cases, it's assumed that you have already installed and configured Shopware on your server, and that it's running properly. This document does not cover configuration options *needed* by Shopware (for example, increasing PHP's `memory_limit` to an acceptable level), and focus only on making an already working system perform better.
```

Comment:
hey @Asabs thanks for stating this :+1: 
I think the author wanted say, that we added new `dependencies` instead of `addictions`. both meaning the same in german :wink: :smiley: 
Does `dependencies` sounds more suitable for you?

Reply:
👍

Reply:
would you like to adjust your PR accordingly?